jsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+  "compilerOptions": {
+    "baseUrl": "./src"
+  }
+}
```

Comment:
Let's add a trailing newline just because I want to be extra-nitpicky even before we have linters :trollface: